DEBUG_COMMANDS.md

```
@@ -55,21 +55,21 @@ Logger.configure(level: :info)
 
 ## Check if specific kills are being processed
 
-With the logging added to RedisQ, you should see info-level messages like:
+With the logging added to R2Z2, you should see info-level messages like:
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)